src/s1reader/s1_burst_slc.py

```diff
@@ -1083,36 +1142,38 @@ def width(self):
 
     @property
     def swath_name(self):
-        '''Swath name in iw1, iw2, iw3.'''
+        """Swath name in iw1, iw2, iw3."""
         return self.burst_id.subswath.lower()
 
     @property
     def thermal_noise_lut(self):
-        '''
+        """
         Returns the LUT for thermal noise correction for the burst
-        '''
+        """
         if self.burst_noise is None:
-            raise ValueError('burst_noise is not defined for this burst.')
+            raise ValueError("burst_noise is not defined for this burst.")
 
         return self.burst_noise.compute_thermal_noise_lut(self.shape)
 
     @property
     def eap_compensation_lut(self):
-        '''Returns LUT for EAP compensation.
+        """Returns LUT for EAP compensation.
 
         Returns:
         -------
             _: Interpolated EAP gain for the burst's lines
 
-        '''
+        """
         if self.burst_eap is None:
-            raise ValueError('burst_eap is not defined for this burst.'
-                            f' IPF version = {self.ipf_version}')
+            raise ValueError(
+                "burst_eap is not defined for this burst."
+                f" IPF version = {self.ipf_version}"
+            )
 
         return self.burst_eap.compute_eap_compensation_lut(self.width)
 
     @property
     def relative_orbit_number(self):
-        '''Returns the relative orbit number of the burst.'''
-        orbit_number_offset = 73 if self.platform_id == 'S1A' else 202
+        """Returns the relative orbit number of the burst."""
+        orbit_number_offset = 73 if self.platform_id == "S1A" else 202
```

Comment:
```suggestion
        if self.platform_id == 'S1A':
            orbit_number_offset = 73
        elif self.platform_id == 'S1B':
            orbit_number_offset = 27
        else:
            orbit_number_offset = 99
```

Reply:
Looks good! I just have a suggestion to add S1C orbit number offset

Reply:
I was holding off on https://github.com/isce-framework/s1-reader/issues/141#issuecomment-2628599516 until we could check if the orbit was fully settled, but we can commit this for now